osu.Game/Rulesets/UI/DrawableRulesetDependenciesProvidingContainer.cs

```diff
@@ -0,0 +1,36 @@
+// Copyright (c) ppy Pty Ltd <contact@ppy.sh>. Licensed under the MIT Licence.
+// See the LICENCE file in the repository root for full licence text.
+
+using osu.Framework.Allocation;
+using osu.Framework.Extensions.ObjectExtensions;
+using osu.Framework.Graphics;
+using osu.Framework.Graphics.Containers;
+
+namespace osu.Game.Rulesets.UI
+{
+    public partial class DrawableRulesetDependenciesProvidingContainer : Container
```

Comment:
Not sure how much sense it makes creating this container for a single usage, but I guess it's fine.